src/androidTest/java/de/dennisguse/opentracks/io/file/importer/ExportImportTest.java

```diff
@@ -234,7 +234,7 @@ public void gpx() {
         // when
         // 1. export
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        trackExporter.writeTrack(outputStream);
+        trackExporter.writeTrack(new Track[]{track}, outputStream);
```

Comment:
Why not overload writeTrack() directly in TrackExporter?
* writeTrack(Track track, OutputStream outputStream);
* writeTrack(Track[] tracks, OutputStream outputStream);

Reply:
it seemed wrong to me to have both exporters implement the very same method but i also did not want to create an abstract class so i just left it as is.
i now added the overload